graph/pkg/service/v0/drives.go

```diff
@@ -710,3 +713,22 @@ func (g Graph) DeleteDrive(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(http.StatusNoContent)
 }
+
+func (g Graph) sortSpaces(req *godata.GoDataRequest, spaces []*libregraph.Drive) []*libregraph.Drive {
```

Comment:
As far as I see this method has nothing to do with the `Graph` object. Can't this just be a `func` then? We do not need the `(g Graph)` receiver.